DemoApp/Resources/Base.lproj/Main.storyboard

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since I removed the filled configuration from the button, Xcode decided relayout views since the size changes slightly. That is why there are so many frame changes. iOS 14 users just get configuration button without any background.